.github/workflows/labeler.yml

```
@@ -0,0 +1,13 @@
+on: pull_request_review
+name: Label approved pull requests
+jobs:
+  labelWhenApproved:
+    name: Label when approved
+    runs-on: ubuntu-latest
+    steps:
+    - name: Label when approved
+      uses: pullreminders/label-when-approved-action@master
+      env:
+        APPROVALS: "1"
```

```suggestion
        APPROVALS: "2"
```

You can make here to 2 if @devarshishimpi wants 🤔

---

I'll do that